### ISSUE TITLE
bogus debug information repeated less than 1ms appart

### ISSUE THREAD

**Issue by kryztoval** [aeternity/epoch#1693]
## Expected Behavior
Getting none duplicated messages almost instantly
...

## Actual Behavior
getting lots of duplicated messages
...

## Steps to Reproduce the Problem

  1. run docker image v0.24.0 
  
  
## Logs, error output, etc.
 
 `2018-10-25 10:59:37.423 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10155 ms for the next connection
2018-10-25 10:59:37.423 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10155 ms for the next connection
2018-10-25 10:59:37.423 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10155 ms for the next connection
2018-10-25 10:59:37.424 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10154 ms for the next connection
2018-10-25 10:59:37.424 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10154 ms for the next connection
2018-10-25 10:59:37.424 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10154 ms for the next connection
2018-10-25 10:59:37.424 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10154 ms for the next connection
2018-10-25 10:59:37.424 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10154 ms for the next connection
2018-10-25 10:59:37.425 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10153 ms for the next connection
2018-10-25 10:59:37.425 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10153 ms for the next connection
2018-10-25 10:59:37.425 [info] <0.1250.0>@aec_peers:on_add_peer:1048 Peer "pp_xpDYTSi...Yva4xN2" - ignoring peer address changed from 187.160.102.31:3015 to 187.160.102.166:3015 by 54.189.139.100
2018-10-25 10:59:37.425 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10153 ms for the next connection
2018-10-25 10:59:37.426 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10152 ms for the next connection
2018-10-25 10:59:37.426 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10152 ms for the next connection
2018-10-25 10:59:37.426 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10152 ms for the next connection
2018-10-25 10:59:37.426 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10152 ms for the next connection
2018-10-25 10:59:37.427 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10152 ms for the next connection
2018-10-25 10:59:37.427 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10151 ms for the next connection
2018-10-25 10:59:37.427 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10151 ms for the next connection
2018-10-25 10:59:37.427 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10151 ms for the next connection
2018-10-25 10:59:37.427 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10151 ms for the next connection
2018-10-25 10:59:37.428 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10150 ms for the next connection
2018-10-25 10:59:37.428 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10150 ms for the next connection
2018-10-25 10:59:37.428 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10150 ms for the next connection
2018-10-25 10:59:37.428 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10150 ms for the next connection
2018-10-25 10:59:37.429 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10149 ms for the next connection
2018-10-25 10:59:37.429 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10149 ms for the next connection
2018-10-25 10:59:37.429 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10149 ms for the next connection
2018-10-25 10:59:37.429 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10149 ms for the next connection
2018-10-25 10:59:37.429 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10149 ms for the next connection
2018-10-25 10:59:37.430 [debug] <0.1250.0>@aec_peers:schedule_connect:723 Waiting 10148 ms for the next connection
`
This will flood the disk with logs so fast

## Specifications

Virtualization: version 18.06.1-ce, build e68fc7a
Hardware specs:
RAM: 4gb
CPU: 2.4GHz 2xCores
OS: Centos 7
Node Version: aeternity/epoch:latest (36c969d29950) 0.24

**Comment by kryztoval** [comment]
I found out there is a PR for a Log level configuration at #841 and it has been completed, but I can't find any examples anywhere on how to use it, and I tried adding "debugging:" and "level: "error"" to the yaml file and it removed the messages from the log files.

**Comment by dincho** [comment]
> I found out there is a PR for a Log level configuration at #841 and it has been completed, but I can't find any examples anywhere on how to use it, and I tried adding "debugging:" and "level: "error"" to the yaml file and it removed the messages from the log files.

I'm not sure what you mean, but to lower the log level you could use:

```
logging:
    level: warning
```

or

```
logging:
    level: error
```

In your configuration file.

**Comment by kryztoval** [comment]
yeah, that is what i did, but it still shows a lot of info messages on docker logs. why does it have to tell me repeatedly how many miliseconds it has to wait several times per milisecond?
